common/spdat.cpp

```diff
@@ -776,6 +776,19 @@ bool IsResurrectionEffects(uint16 spell_id)
 	return false;
 }
 
+int8 GetResurrectionSicknessCheck(uint16 spell_id1, uint16 spell_id2)
+{
+	if (RuleI(Spells, ResurrectionEffectsBlock) == RES_EFFECTS_BLOCK) {
+		LogSpells("ResurrectionEffectsBlock triggered -- [{}] is blocked by [{}]", spells[spell_id2].name, spells[spell_id1].name);
+		return -1;	// can't stack
+	}
+	else if (RuleI(Spells, ResurrectionEffectsBlock) == RES_EFFECTS_BLOCK_WITH_BUFFS) {
+		LogSpells("ResurrectionEffectsBlock triggered -- [{}] is blocked by [{}], moving to empty slot if available", spells[spell_id2].name, spells[spell_id1].name);
+		return 2;	// move to empty slot if available
```

Comment:
Why aren't we using the constants in the return here? We try to make a concerted effort to eliminate magic numbers for readability when we can